addons/project/views/project_views.xml

```diff
@@ -1199,10 +1191,10 @@
                                     <field name="allow_milestones" invisible="1"/>
                                     <field name="parent_id" invisible="1" />
                                     <field name="is_closed" invisible="1" />
-                                    <field name="priority" widget="priority" optional="show" nolabel="1"/>
-                                    <field name="state" widget="project_task_state_selection" optional="show" nolabel="1"/>
-                                    <field name="child_text" invisible="1"/>
-                                    <field name="allow_subtasks" invisible="1"/>
```

Comment:
We should keep them invisible on the view to avoid breaking custo code, in the case they did a xpath on those fields. 🙂

Reply:
Done